### ISSUE TITLE
Link to Actor Macro "Token Says " not working.

### ISSUE THREAD

**Issue by Steelskinz** [napolitanod/Token-Says#124]
Hi, so I was trying to configure an actor to say a sentence when I push on a macro, I followed up the doc like we should.

This is when the token is linked, it works like it should. 
![image](https://github.com/napolitanod/Token-Says/assets/93941729/10197289-9b8f-4fb2-90c6-202163c0ecfc)

```
const options = {
    audio: {
        compendium: '',
        source: '',
        quote: ''
    },
    chat: {
        compendium: '',
        source: '',
        quote: 'Hello World!'
    },
    delay: 0,
    lang: '',
    likelihood: 100,
    suppress: {
        bubble: false,
        message: false,
        quotes: false
    },
    volume: 0.50
}

tokenSays.saysDirect('4vVqmRRftykuIIjz', null, null, options);
```

However if I changed to : 
```
const options = {
    audio: {
        compendium: '',
        source: '',
        quote: ''
    },
    chat: {
        compendium: '',
        source: '',
        quote: 'Hello World!'
    },
    delay: 0,
    lang: '',
    likelihood: 100,
    suppress: {
        bubble: false,
        message: false,
        quotes: false
    },
    volume: 0.50
}

tokenSays.saysDirect('', 'I0HIwGdMAVUzIO6b', '', options);
```
It will not work, no error appeared in the console, and yes, actor data is linked, i'm using PF2E and the last V10 version up to date (Did not upgrade to V11 yet.) 
I asked on #macro-polo but even there we did not make it work properly.

The reason i'm trying to use an actor instead of a token is if this player happen to delete his character (Or I, because i'm clumsy 69) and I would like not to edit macros on the fly in the middle of the session. 

Thanks for reading me

**Comment by napolitanod** [comment]
I see where the issue is. I need to test the fix to make sure I don't mess something else out and will push it out once ready.

**Comment by napolitanod** [comment]
Right now this is available in 1.12.1 but that requires v11. I don't have a means for testing v10. I'll try to get the fix into v10 branch when I get a sec, tho won't be able to test it.

**Comment by Steelskinz** [comment]
I've updated to V11, Macro is working correctly, thanks for your patience.
